### DISCUSSION TITLE
Cannot Login Account

### DISCUSSION THREAD

**Discussion by fauzanhlmy** [community/community#146122] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

Dear github support, please help me to reach my first account github.
my problem is :
- i know my username
- i know my password
- i know my email
- i lost my 2FA auth (due my resignation on my latest devices)
- i lost my recovery code  (due my resignation on my latest devices)
- i doesn't have my passkey.
Please help me because a lot of private project been there. I'm trying to reaching out github support but it always bot chat that communicated with me.

**Comment by quoc10973** [comment]
To recover your GitHub account when facing issues with 2FA and not having your recovery codes, you should follow these steps:

Visit GitHub Support: Go to GitHub’s support page at [GitHub Support](https://support.github.com/contact).

Select "Account Recovery": GitHub provides a process for recovering accounts for users facing issues with two-factor authentication (2FA). You will be asked to provide account information like your username, email, and a description of the problem.

Fill out the support request form: Provide detailed information about your situation, including:

Your GitHub username.
The email associated with your account.
Explain that you've lost access to your 2FA methods and recovery codes.
Mention any steps you've already taken (like contacting the bot, etc.).
Submit the request: After filling out the form, submit the request and wait for a response from GitHub.

Contact via email (if needed): If you don’t get a response from the support form, you can also try emailing GitHub directly at support@github.com with similar details.

Please note that the recovery process may take some time as GitHub needs to verify your information to protect your account. If possible, provide any additional details that GitHub may require to prove ownership of the account.

Good luck, and I hope you're able to regain access to your account!

Mark this answered if you see this helpful for you

**Comment by queenofcorgis** [comment, marked as answer]
Hi there, 

Thank you for your patience. The community is unable to assist with nor escalate with this account-related situation. Please review our doc: [Recovering your account if you lose your 2FA credentials](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials). Within the doc, you also have the option to go through recovery with a Virtual Agent. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
